### ISSUE TITLE
beempy power gives divide by zero error for accounts with 0 SP

### ISSUE THREAD

**Issue by steemsidequest** [holgern/beem#87]
Hello,

In playing around with beempy this morning I found that since accounts can be created with 0 SP, using beempy to look up account power can lead to division by zero:

```
$ beempy power scms-sc

@scms-sc
Traceback (most recent call last):
  File "/Environments/alpha/bin/beempy", line 11, in <module>
    sys.exit(cli())
  File "/Environments/alpha/lib/python3.6/site-packages/click/core.py", line 764, in __call__
    return self.main(*args, **kwargs)
  File "/Environments/alpha/lib/python3.6/site-packages/click/core.py", line 717, in main
    rv = self.invoke(ctx)
  File "/Environments/alpha/lib/python3.6/site-packages/click/core.py", line 1163, in invoke
    rv.append(sub_ctx.command.invoke(sub_ctx))
  File "/Environments/alpha/lib/python3.6/site-packages/click/core.py", line 956, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/Environments/alpha/lib/python3.6/site-packages/click/core.py", line 555, in invoke
    return callback(*args, **kwargs)
  File "/Environments/alpha/lib/python3.6/site-packages/beem/cli.py", line 974, in power
    a.print_info(use_table=True)
  File "/Environments/alpha/lib/python3.6/site-packages/beem/account.py", line 320, in print_info
    vote_mana = self.get_manabar()
  File "/Environments/alpha/lib/python3.6/site-packages/beem/account.py", line 403, in get_manabar
    estimated_pct = estimated_mana / estimated_max * 100
ZeroDivisionError: division by zero
```
I'm sorry I don't know the fix (adding some prettier error handling code somewhere?), I don't know much python.

**Comment by crokkon** [comment]
which beem version are you using? This should be fixed with [0.20.4](https://github.com/holgern/beem/commit/39476777d25e6bf6cd2337a10ff03f75c8cfe096)
```
# beempy power scms-sc

@scms-sc
+-----------------------------+------------------------+
| Key                         | Value                  |
+-----------------------------+------------------------+
| Name (rep)                  | scms-sc (25.00)        |
| Voting Power                | 24.69 %,               |
| Vote Value                  | 0.00 $                 |
| Last vote                   | 427310:27:57 ago       |
| Full in                     | 90:22:29               |
| Steem Power                 | 0.00 STEEM             |
| Balance                     | 0.000 STEEM, 0.000 SBD |
| Remaining RC                | 100.00 %               |
| Remaining RC                | (6 G RC of 6 G RC)     |
| Full in                     | 0:00:00                |
| Est. RC for a comment       | 1.51 G RC              |
| Est. RC for a vote          | 0.31 G RC              |
| Est. RC for a transfer      | 0.12 G RC              |
| Est. RC for a custom_json   | 0.18 G RC              |
| Comments with current RC    | 4 comments             |
| Votes with current RC       | 19 votes               |
| Transfer with current RC    | 50 transfers           |
| Custom_json with current RC | 33 transfers           |
+-----------------------------+------------------------+
```

**Comment by steemsidequest** [comment]
Ah! I was using 0.20.3, I didn't realise it was updated since.

Thanks.
